src/ec/clevo/it5570/acpi/rgb_kb.asl

```diff
@@ -0,0 +1,105 @@
+Name (KBEN, 0x01) // Keyboard Backlight Enable
```

Comment:
license header missing. It would be good to issue `make test-lint` when issuing a PR